### PR TITLE
Fix docker compose running problem & running in circleci

### PR DESCRIPTION
**Pull request by ipdae**
solved #106

### CODE REVIEW

File: nekoyume/models.py [hunk 1]
```diff
@@ -327,11 +327,11 @@ def broadcast(self,
         """
         It broadcast this block to every nodes you know.
 
-       :param       sent_node: sent :class:`nekoyume.models.Node`.
-                               this node ignore sent node.
-       :param         my_node: my :class:`nekoyume.models.Node`.
-                               received node ignore my node when they
-                               broadcast received object.
+        :param       sent_node: sent :class:`nekoyume.models.Node`.
```
Comment (nekoyume-ops) [inline nekoyume/models.py]: are these changes necessary?

File: nekoyume/models.py [hunk 2]
```diff
@@ -476,13 +475,13 @@ class Move(db.Model):
     __table_args__ = (
         db.CheckConstraint(
             db.func.lower(user_address).like('0x%') &
-            (char_length(user_address) == 42)
+            (db.func.length(user_address) == 42)
             # TODO: it should has proper test if 40-hex string
```
Comment (longfin) [inline nekoyume/models.py]: why did you change this?
